dotCMS/src/integration-test/java/com/dotcms/concurrent/DotConcurrentFactoryTest.java

```diff
@@ -10,6 +12,163 @@
 
 public class DotConcurrentFactoryTest {
 
+    @Ignore
+    @Test
+    public void testDefaultOne_Submitter_Config() throws JSONException{
+
+        final String submitterName = "testsubmitter";
+        final DotConcurrentFactory dotConcurrentFactory =
+                DotConcurrentFactory.getInstance();
+
+        final DotSubmitter submitter =
+                dotConcurrentFactory.getSubmitter(submitterName,
+                        new SubmitterConfigBuilder().poolSize(2)
+                        .maxPoolSize(4).queueCapacity(500).build()
+                );
+
+        System.out.println(submitter);
+
+        IntStream.range(0, 40).forEach(
+                n -> {
+
+                    if (n % 10 == 0) {
+
+                        System.out.println(submitter);
+                    }
+                    submitter.execute(new PrintTask("Thread" + n));
```

Comment:
Codacy found an issue: [The String literal "Thread" appears 4 times in this file; the first occurrence is on line 35](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6265363)